docs/how-to/vector-stores/sparse-vectors.md

```diff
@@ -0,0 +1,114 @@
+# Using Sparse Vectors with Vector Stores
+
+Ragbits supports sparse embeddings through the `SparseVector` and `SparseEmbedder` classes. This guide explains how to use sparse embeddings with vector stores.
+
+## What are Sparse Embeddings?
+
+Unlike dense embeddings (which are represented as arrays of floating-point numbers), sparse embeddings only store non-zero values and their positions. This makes them more efficient for certain types of data and retrieval tasks.
+
+Sparse embeddings are particularly useful for:
+- Lexical search (keyword matching)
+- Representing large vocabularies efficiently
+- Hybrid search approaches combining dense and sparse representations
+
+## Using Sparse Embeddings with Vector Stores
+
+Ragbits vector stores (Qdrant and InMemory) support both dense and sparse embeddings. Here's how to use them:
+
+### 1. Create a Sparse Embedder
+
+First, create a sparse embedder:
+
+```python
+from ragbits.core.embeddings.sparse import SparseEmbedder
+
+# Example sparse embedder configuration
+sparse_embedder = SparseEmbedder(
```

Comment:
`SparseEmbedder` is an abstract class - you need a specific instance. It also doesn't have the arguments used here.